yarn-project/simulator/src/public/side_effect_trace.ts

```diff
@@ -331,10 +338,36 @@ export class PublicSideEffectTrace implements PublicSideEffectTraceInterface {
     );
   }
 
+  public traceEnqueuedCall(
+    /** The trace of the enqueued call. */
+    _enqueuedCallTrace: this,
+    /** The call request from private that enqueued this call. */
+    _publicCallRequest: PublicCallRequest,
+    /** The call's calldata */
+    _calldata: Fr[],
+    /** Did the call revert? */
+    _reverted: boolean,
+  ) {
+    throw new Error('Not implemented');
+  }
+
+  public traceExecutionPhase(
+    /** The trace of the enqueued call. */
+    _appLogicTrace: this,
+    /** The call request from private that enqueued this call. */
+    _publicCallRequests: PublicCallRequest[],
+    /** The call's calldata */
+    _calldatas: Fr[][],
+    /** Did the any enqueued call in app logic revert? */
+    _reverted: boolean,
+  ) {
+    throw new Error('Not implemented');
+  }
+
   /**
```

Comment:
This old trace doesn't operate at the enqueued call / tx level. It will be removed eventually!